packages/terra-tag/src/RollupTag.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useRef } from 'react';
+import { injectIntl } from 'react-intl';
+import { KEY_SPACE, KEY_RETURN } from 'keycode-js';
+import { v4 as uuidv4 } from 'uuid';
+import classNamesBind from 'classnames/bind';
+import PropTypes from 'prop-types';
+import ThemeContext from 'terra-theme-context';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
+import styles from './Tag.module.scss';
```

Comment:
This new RollupTag component should have its own style module.

Reply:
Changed Done .Thanks